### ISSUE TITLE
[Enhancement]: unify collections and playlists

### ISSUE THREAD

**Issue by wtanksleyjr** [advplyr/audiobookshelf#1262]
### Describe the feature/enhancement

I built a bunch of collections, then discovered playlists (or upgraded to a version that has them, I'm not sure) and moved my personal collections into playlists. Yay! I like them both.

But so far as I can see, the key difference between them is that collections are public and only creatable by an admin, while playlists are private and anyone can make them. I'd like to make them the "same thing", differing only by a toggle switch that an admin can flick; and maybe that's good, and I'd like to offer it for consideration.

But even if that's not good, I'd like to remove the split between them in the user interface, and display both on the same page, so that I can just look at all the collections _and_ all of my playlists at the same time, and add/remove to/from them from the same modal dialogue, reducing the number of menu items and buttons.

One side effect: this will reduce the number of tab icons down the side of the home page. On my main laptop (which is tiny at 11") I can no longer click on the "problems" button when something goes wrong, there are just too many tabs and the version info gets in the way. This isn't really a "fix" or a "bug", since no matter what something else needs to be done, but it will forestall the day of reckoning.

**Comment by advplyr** [comment]
Playlists were just recently added and being worked on for the mobile apps.

Collections are only for books but playlists are also supported for podcast episodes. A feature will be added to make a playlist from a collection or a player queue. Playlists will also be for music when that time comes.

They aren't going to be merged and it sounds like you only need to use collections. If you only use collections then you won't see the Playlists button on the side at all.

I did notice some bugs with the siderail overflowing.

**Comment by wtanksleyjr** [comment]
Thank you for an awesome app. I won't respond again lest I waste your time,
but I don't understand your answer at all.

But I do use both playlists and collections. That's why I want to see them
both on the same screen and get to them from the same button. I wouldn't
have thought of that if I didn't use both.

I'm glad playlists can contain both episodes and books, but I don't see
that as a reason they can't or shouldn't be displayed on the same
page/modal. You don't have to answer, but I have no idea why that matters.
I'm not asking for them to be merged in the code, only for the UI to allow
for viewing them all under one page and/or adding books to them under one
modal.

But again, I promise not to debate.
